project_category/models/project_type.py

```diff
@@ -0,0 +1,35 @@
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import _, api, fields, models
+from odoo.exceptions import ValidationError
+
+
+class ProjectType(models.Model):
+    _name = "project.type"
+    _description = "Project Type"
+    _rec_name = "complete_name"
+
+    parent_id = fields.Many2one(comodel_name="project.type", string="Parent Type")
+    child_ids = fields.One2many(
+        comodel_name="project.type", inverse_name="parent_id", string="Subtypes"
+    )
+    name = fields.Char(required=True, translate=True)
+    complete_name = fields.Char(compute="_compute_complete_name", store=True)
```

Comment:
```suggestion
    complete_name = fields.Char(
        compute="_compute_complete_name", 
        recursive=True,
        store=True
    )
```

Sorry. I missed the error. @AaronHForgeFlow  showed the error correctly. There is suggestion :slightly_smiling_face:

Reply:
@AaronHForgeFlow and @Mantux11 Done